sapi/fpm/fpm/fpm_conf.c

```diff
@@ -1497,11 +1504,28 @@ static void fpm_conf_ini_parser_array(zval *name, zval *key, zval *value, void *
 	char *err = NULL;
 	void *config;
 
-	if (!*Z_STRVAL_P(key)) {
-		zlog(ZLOG_ERROR, "[%s:%d] Misspelled  array ?", ini_filename, ini_lineno);
+	if (
+		zend_string_equals_literal(Z_STR_P(name), "access.suppress_path")
+	) {
+		if (!(*Z_STRVAL_P(key) == '\0')) {
+			zlog(ZLOG_ERROR, "[%s:%d] Keys provided to field 'access.suppress_path' are ignored", ini_filename, ini_lineno);
+			*error = 1;
+		}
+		if (!(*Z_STRVAL_P(value)) || (*Z_STRVAL_P(value) != '/')) {
+			zlog(ZLOG_ERROR, "[%s:%d] Values provided to field 'access.suppress_path' must begin with '/'", ini_filename, ini_lineno);
+			*error = 1;
+		}
+		if (*error) {
+			return;
+		}
+	} else if (
+		!*Z_STRVAL_P(key)
+	) {
```

Comment:
```suggestion
	} else if (!*Z_STRVAL_P(key)) {
```

Reply:
yeah, you're right here, just a side effect from rebasing on top of 312fa05a291bdf17e605aab28d73a258e3c2c2b2 without engaging my brain (which is also why I hadn't just fixed that at the time, cargo culting on how it was written before)